commons/src/main/java/org/archive/util/BloomFilter64bit.java

```diff
@@ -80,7 +80,7 @@ public BloomFilter64bit( final long n, final int d, boolean roundUp) {
      * @param roundUp if true, round bit size up to next-nearest-power-of-2
      */
     public BloomFilter64bit(final long n, final int d, Random weightsGenerator, boolean roundUp ) {
```

Comment:
I'd love to remove the unused `Random` parameter, but I'm afraid it'd break clients unknown. What do you think about deprecating this constructor, as well as the methods that are seemingly unnecessary and used only in the tests (e.g. https://github.com/internetarchive/heritrix3/blob/master/commons/src/main/java/org/archive/util/BloomFilter64bit.java#L157 & https://github.com/internetarchive/heritrix3/blob/master/commons/src/test/java/org/archive/util/BloomFilterTest.java#L159) to further clean this up?

Reply:
Not a bad idea, but I'm not sure if the deprecation warnings show up when you're tying things together with Spring?

Reply:
Well, either way, yes, please mark that as deprecated.

Reply:
okay fine let's take this as-is then.